modules/persistent-storage-csi-snapshots-delete.adoc

```diff
@@ -26,4 +23,65 @@ deletionPolicy: Delete <1>
 ----
 <1> If the `Delete` value is set, the underlying snapshot will be deleted, along with the `VolumeSnapshotContent` object. If the `Retain` value is set, both the underlying snapshot and `VolumeSnapshotContent` object remain.
```

Comment:
A little strange for this sentence, should it be like this?
When deleting the `VolumeSnapshot` object: If the `Delete` value is set, the underlying snapshot will be deleted, along with the `VolumeSnapshotContent` object. If the `Retain` value is set, both the underlying snapshot and `VolumeSnapshotContent` object remain.

Reply:
Good point, I've added your suggestion.